### ISSUE TITLE
moose.doc( 'moose.Compartment' ) does not work

### ISSUE THREAD

**Issue by dilawar** [BhallaLab/moose-core#204]
`mosoe.doc` does not work.

```
>>> moose.doc( 'moose.Compartment' )
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/dist-packages/moose/moose.py", line 503, in doc
    text += getmoosedoc(tokens, inherited=inherited)
  File "/usr/lib/python2.7/dist-packages/moose/moose.py", line 408, in getmoosedoc
    class_element = _moose.element('/classes/%s' % (tokens[0]))
NameError: global name '_moose' is not defined
```

Affects both `chamcham` and `master` branch.

**Comment by dilawar** [comment]
Fixed by #205 .

- [ ] Apply patch to `chamcham` branch.

**Comment by dilawar** [comment]
#209 fixes this.
